tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -20,11 +20,16 @@
 import './flags_webgpu';
 
 import {backend_util, DataStorage, DataType, engine, env, findBackend, KernelBackend, Rank, RecursiveArray, ShapeMap, slice_util, Tensor, Tensor2D, Tensor3D, Tensor4D, TimingInfo, util} from '@tensorflow/tfjs-core';
-import {Glslang} from '@webgpu/glslang/dist/web-devel/glslang.onefile';
 // TODO(xing.xu): use FusedConv2DConfig from backend_util:
 // https://github.com/tensorflow/tfjs/issues/2471
 // tslint:disable-next-line: no-imports-from-dist
 import {FusedConv2DConfig} from '@tensorflow/tfjs-core/dist/ops/fused_util';
+// TODO TODO to import computeOptimalWindowSize and other reduce_util function
+// from backend_util, rather than from 'dist'
+import {assertAxesAreInnerMostDims, computeOutAndReduceShapes} from '@tensorflow/tfjs-core/src/ops/axis_util';
+import {computeOptimalWindowSize} from '@tensorflow/tfjs-core/src/ops/reduce_util';
+import {sumOutType} from '@tensorflow/tfjs-core/src/types';
```

Comment:
Please edit `tfjs-core/src/index.ts:71` to be: `export {DataType, DataTypeMap, DataValues, Rank, RecursiveArray, ShapeMap, TensorLike, sumOutType} from './types';`. Then add a TODO comment above this line to import `sumOutType` directly from '@tensorflow/tfjs-core'.